### DISCUSSION TITLE
Issue with the tint of progressView and button

### DISCUSSION THREAD

**Discussion by leogiroux** [LeoNatan/LNPopupController#453] [General] [answered]
Hello @LeoNatan 

There is a little bug with the tint of progressView and button when presenting a UIViewController modally.
`appearanceProxy.tintColor = .yellow` is not set and the color is blue by default instead of the yellow.

This is the appearanceProxy init inside an UITabBarController
<img width="1012" alt="Capture d’écran 2021-06-03 à 11 19 30" src="https://user-images.githubusercontent.com/12122348/120620789-996d1e80-c45d-11eb-9497-dbdc8412ac96.png">

This how I present the controller with the bug
<img width="520" alt="Capture d’écran 2021-06-03 à 11 21 11" src="https://user-images.githubusercontent.com/12122348/120621016-cde0da80-c45d-11eb-9562-6dea610eed68.png">

this the expected result
<img width="348" alt="Capture d’écran 2021-06-03 à 11 21 37" src="https://user-images.githubusercontent.com/12122348/120621078-ddf8ba00-c45d-11eb-85c0-d54f609c2382.png">

This is the controller with the bug
<img width="359" alt="Capture d’écran 2021-06-03 à 11 30 52" src="https://user-images.githubusercontent.com/12122348/120622722-5ca22700-c45f-11eb-9cd8-3c247a24ba01.png">

Thanks for your help :)

**Comment by LeoNatan** [comment, marked as answer]
You are using the appearance API incorrectly. `appearanceProxy.progressView.tintColor` is not a correct syntax that the UIKit system can work with. Instead, you should create an appearance proxy for the progress view, and set that tint color.

```swift
let progressViewAppearanceProxy = UIProgressView.appearance(whenContainedInInstancesOf: [LNPopupBar.self, UIViewController.self])
progressViewAppearanceProxy.tintColor = .white
```
